.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "import/extensions": [ 1, {
+        "js": "always", "json": "always"
+      }]
+    },
+    "ignorePatterns": [
+      "dist/",
+      "build/"
+    ]
+  }
+  
```

Comment:
- [ ] in this project, it would be better to set up both [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters in the project repository Currently, You are checking linters twice Kindly set your linters correctly, you may check in ```.github/workflows``` folder 😂 
  
![image](https://user-images.githubusercontent.com/102757126/217046868-625c6d3e-95dd-40aa-9856-1a10c3ce1a28.png)


   _Remember, setting up linters in a project repository is a good practice to ensure the code is written following a consistent style and adheres to best practices. JavaScript and HTML&CSS linters can help identify syntax errors, typo errors, and other issues that might negatively impact the quality and functionality of the code. 👍👍_